docs/user/reference/metrics/custom_distribution.md

```diff
@@ -87,21 +112,57 @@ import org.mozilla.yourApplication.GleanMetrics.Graphics
 val snapshot = Graphics.checkerboardPeak.testGetValue()
 
 // Does the sum have the expected value?
-assertEquals(11, snapshot.sum)
+assertEquals(23, snapshot.sum)
 
 // Usually you don't know the exact timing values, but how many should have been recorded.
-assertEquals(2L, snapshot.count())
+assertEquals(1L, snapshot.values.size)
+```
+
+</div>
+
+<div data-lang="Java" class="tab">
+
+```Java
+import org.mozilla.yourApplication.GleanMetrics.Graphics;
+
+// Get snapshot
+val snapshot = Graphics.INSTANCE.checkerboardPeak().testGetValue();
+
+// Does the sum have the expected value?
+assertEquals(23, snapshot.sum);
 ```
 
 </div>
+<div data-lang="Swift" class="tab">
+
+```Swift
+// Get snapshot.
+let snapshot = graphics.checkerboardPeak.testGetValue()
 
-<div data-lang="Java" class="tab"></div>
-<div data-lang="Swift" class="tab"></div>
-<div data-lang="Python" class="tab"></div>
+// Usually you don't know the exact timing values,
```

Comment:
timing? This is a custom distribution, sir.

Reply:
coooopy-pasta.